packages/blade/src/components/Tabs/Tabs.stories.tsx

```diff
@@ -12,42 +12,19 @@ import { Box } from '~components/Box';
 export default {
```

Comment:
RN stories are very empty right now. Can you at least add some basic stories for variants & sizes? Or will that be done in documentation PR?

Reply:
Yes more stories are added in the documentation PR. Check #1715